app/src/main/res/layout/view_confirm_resend_panel.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge
```

Comment:
merge doesn't really make sense here, right? a normal include would work just as well.

Reply:
yes, actually both tags are unnecessary - just having a root view group is enough